### PR TITLE
Vertical navigation to site

### PR DESCRIPTION
**Pull request by mikearildbrown**

### CODE REVIEW

File: site/docs/patterns/vertical-navigation.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+data:
+  components: ["Navigation item"]
+  resources:
+    [
+      { href: "https://github.com/jpmorganchase/salt-ds/issues?q=is%3Aopen+is%3Aissue+label%3A%22pattern%3A+navigation%22+label%3A%22type%3A+enhancement+%F0%9F%92%A1%22", label: "Navigation enhancements", internal: false, resourceType: "github" },
+      { href: "https://www.figma.com/file/AcRj2QE1ZEiTrx7C3K0O92/Navigation-Pattern-Templates?type=design&node-id=15%3A598&mode=design&t=Bul9WEJNrXM1X2Xv-1", label: "Navigation examples", internal: true },
+    ]  
+---
+
+Vertical navigation provides users with a structured and intuitive way to navigate different pages, sections or features of your website or application. It's often on the left of the viewport and organizes links into a single- or multi-level structure.
```
Comment (joshwooding) [inline site/docs/patterns/vertical-navigation.mdx:15]:
```suggestion
Vertical navigation provides users with a structured and intuitive way to navigate different pages, sections or features of your website or application. It's often on the left of the viewport and organizes links into a single or multi-level structure.
```
?

File: site/docs/patterns/vertical-navigation.mdx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+## How to use
+
+### Anatomy
+
+A vertical navigation consists of a stack of navigation item components:
+
+1. [Default navigation item](/salt/components/navigation-item/examples#vertical): When clicked on, a navigation item links the user to a different section of a website or application. Only one navigation item can be active at a single time.
+2. [Parent navigation item](/salt/components/navigation-item/examples#vertical-nested-group): Displayed with a chevron and is expandable only. Upon interaction a parent navigation item doesn’t direct the user anywhere and, instead, reveals reveals nested navigation items inside it. 
+
+You'll typically use vertical navigation menus in a sidebar, on the left side of the screen, within a [drawer](/salt/components/drawer), or a [parent-child layout](/salt/components/parent-child-layout).
```
Comment (joshwooding) [inline site/docs/patterns/vertical-navigation.mdx:32]: Is menus the right term here?

**Comment by joshwooding** [comment]
I've left comments to @navkaur76 in the Figma file

**Review by origami-z — COMMENTED** [review]
Do we need to move this together with "Navigation" pattern?

Also, anything we can update for Navigation Item component doc?

**Comment by origami-z** [comment]
500 Error
https://saltdesignsystem-git-vertical-nav-to-site-fed-team.vercel.app/salt/patterns/navigation/index

**Review by origami-z — APPROVED** [review]
This is the first pattern having examples code in relevant component pages. We should discuss what's the way forward.

**Review by joshwooding — APPROVED** [review]

**Pull request merged**